VRDR/IJEMortality.cs

```diff
@@ -4411,29 +4421,49 @@ public string DMIDDLE
         [IJEField(167, 1858, 50, "Father's First Name", "DDADF", 1)]
         public string DDADF
         {
+            // TODO: Implement mapping from FHIR record location: DecedentFather
             get
             {
-                // TODO: Implement mapping from FHIR record location: DecedentFather
+                string[] names = record.FatherGivenNames;
+                if (names != null && names.Length > 0)
+                {
+                    return names[0];
+                }
                 return "";
             }
             set
             {
-                // TODO: Implement mapping to FHIR record location: DecedentFather
+                if (!String.IsNullOrWhiteSpace(value))
+                {
+                    record.FatherGivenNames = new string[] { value.Trim() };
+                }
             }
         }
 
         /// <summary>Father's Middle Name</summary>
         [IJEField(168, 1908, 50, "Father's Middle Name", "DDADMID", 1)]
         public string DDADMID
```

Comment:
middle names should have priority = 2 instead of 1 to make sure the given names array is constructed in the correct order